stable/coredns/templates/service.yaml

```diff
@@ -26,4 +26,4 @@ spec:
   {{- end }}
   ports:
 {{ include "coredns.servicePorts" . | indent 2 -}}
-  type: {{ default "ClusterIP" .Values.serviceType }}
+  type: {{ default "ClusterIP" .Values.service.type }}
```

Comment:
This is a breaking change, so needs to be bumped to v 2.     also I don't see a change to values.yaml to represent the change.

Reply:
I've updated values.yaml to reflect the change as well as bumping version number to 2.0.0